### DISCUSSION TITLE
No overview page for the keywords

### DISCUSSION THREAD

**Discussion by Maik-Wi** [adityatelange/hugo-PaperMod#1578] [Q&A] [answered]
Since I switched to Hugo v0.133.0, the page for the keywords is no longer generated. Both on macOS and on Debian. I mean the side http://localhost:1313/tags/

Do others feel the same way?

**Comment by jasonbraganza** [comment]
working here, @Maik-Wi 
on linux mint with
`hugo v0.133.0-c9777473d1369f812d727a6c07dc57ad7be7bf62+extended linux/amd64 BuildDate=2024-08-17T19:57:41Z VendorInfo=gohugoio`

![image](https://github.com/user-attachments/assets/2a5f4e73-2e36-459b-b9a2-af54aa08dbff)

**Reply by Maik-Wi** [reply]
Thank you for your feedback and your test. 

Do I correctly assume that you are using the [version in your repository](https://github.com/jasonbraganza/hugo-PaperMod)? I just cloned them and threw them in my content folder. I don't see the tag page there either.

I use almost the same version as you
`hugo v0.133.0+extended darwin/arm64 BuildDate=2024-08-17T19:57:41Z VendorInfo=brew`

This indicates that the way I use buzzwords seems to be different. My YAML header looks like this.

```
---
title: "Title"
date: 2024-08-24T10:05:53+02:00
layout: post
author: "maik"
tags: ["tag1", "tag2", "tag3", "tag4", "tag5", "tag6", "tag7"]
---
```

Can you still share with me what yours looks like?

Thank you in advance for your time.

**Reply by jasonbraganza** [reply]
oooooh! i didn’t think of the Papermod version! No I’m running an older version.
will try this next week and report back.
If you or others here find something earlier, then that’d be even better!

**Comment by jasonbraganza** [comment]
@Maik-Wi, got to locally work with the latest `master` branch of Papermod. Seems to work, even with that.

I recently changed my sections, but it's still the same sort of yaml as before.

```
---
title: "TIL: The Difference Between a CLI Tool and a Freeze Tool"
date: 2023-03-01T11:35:47+05:30
categories: ["work"]
tags: [dgplug, python, planetpython, linux, mjb-what-i-learned]
---
```

I think the main difference is that I am not specifiying any layout. (just letting it default to whatever the theme specifies)

![Tag page](https://github.com/user-attachments/assets/2766b5f8-73bc-4921-b907-2c08d9d3b642)

**Comment by Maik-Wi** [comment, marked as answer]
I was able to fix the issue. I had used Unicode symbols in the tags, specifically ★ as a tag. This caused the tag pages to not render correctly. So, the issue wasn’t with the theme but with the CMS itself.

**Reply by jasonbraganza** [reply]
Thank you for this!
Even I use unicode a lot. The only reason they aren’t in my tags is pure chance 😂
Will take care to avoid them
